src/testers.jl

```diff
@@ -97,9 +97,14 @@ function _make_jvp_call(fdm, f, xs, ẋs, ignores)
     ignores = collect(ignores)
     all(ignores) && return ntuple(_->nothing, length(xs))
     sigargs = zip(xs[.!ignores], ẋs[.!ignores])
-    return jvp(fdm, f2, sigargs...)
+    return to_composite(jvp(fdm, f2, sigargs...))
 end
 
+# remove after https://github.com/JuliaDiff/FiniteDifferences.jl/issues/97
+to_composite(x::Tuple) = Composite{typeof(x)}(x...)
+to_composite(x::NamedTuple) = Composite{typeof(x)}(;x...)
+to_composite(x) = x
```

Comment:
Since they are not expored
```suggestion
_to_composite(x::Tuple) = Composite{typeof(x)}(x...)
_to_composite(x::NamedTuple) = Composite{typeof(x)}(;x...)
_to_composite(x) = x
```

Might want to say:

```suggestion
_maybe_as_composite(x::Tuple) = Composite{typeof(x)}(x...)
_maybe_as_composite(x::NamedTuple) = Composite{typeof(x)}(;x...)
_maybe_as_composite(x) = x
```
or something?

Reply:
or maybe including `fix` in the name?

Reply:
all good suggestions, so added them all